### ISSUE TITLE
In mobile, screen reader focus moves to the search feature as though it were selected and open, even though visually it is not

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#2503]
## Source

Pacific ADA

## Describe the bug

In the mobile view, after navigating to the bottom of the page, with a screen reader focus moves to the search feature as though it were selected and open, even though visually it is not.   See video.

## Screenshots

https://github.com/proudcity/wp-proudcity/assets/1569463/92638f35-be50-4402-9663-6a4b5b9e3e59

**Comment by curtismchale** [comment]
We've looked at revamping the search anyway. I wonder if this should be bundled as a requirement for a revamped search dialogue?

**Comment by lukefretwell** [comment]
@curtismchale probably. Note I added a search label to help us group issues better (outside of milestones).
